### ISSUE TITLE
GG Max Feature Request - Light connectable by Switch unlimited or increased

### ISSUE THREAD

**Issue by UltraVox001** [Dark-Basic-Software-Limited/GameGuruRepo#2175]
Currently, Max does not allow more than 10 Lights to be connected to a switch. It's not much. This is fine for an ordinary home, but when it comes to powering a big power plant, an imposing factory, a military base or a giant structure, the 10 Lights are no longer enough. In addition, it limits the scenarios considerably.

I would like to see this limit of 10 Lights per switch increased significantly. Let it go up to 50, or let that limit disappear. If my game scenario requires massive lighting from a single switch, I must be able to do so. Max must not interfere by limiting me or offering me restricted choices.

A way around this problem would be to add to the script switches the possibility of turning on others, themselves connected to Lights off (SwitchToSwitch).
![SwitchToSwitch](https://user-images.githubusercontent.com/96718175/154534568-45953587-2dc1-43d3-9441-fbd9e2ae4f48.png)
I am attaching a test map to this post.
[SwitchToSwitch.zip](https://github.com/TheGameCreators/GameGuruRepo/files/8090750/SwitchToSwitch.zip)
You can see that it is not possible to command a switch A to turn on a switch B which is connected to 9 lights.
If that had been the case, the problem would have been solved (by workaround).

**Comment by ArgentArts** [comment]
One of the first things I tried was to daisy-chain several lights together to see if turning off one would toggle the next down the line, but it didn't, which is too bad. If we could daisy-chain them (connect a switch to a light and that light to the next light, etc.) then it could be unlimited.

**Comment by UltraVox001** [comment]
It was the first thing I tried the first time I had Max in my hands, months ago. :)
I was disappointed that it didn't work that way.

**Comment by ArgentArts** [comment]
We need a HIDE function, it seems. That way we could HIDE the wall in order to connect the switch to the light and then unhide the wall when completed.

**Comment by UltraVox001** [comment]
This HIDE mode should be active on all non-connectable elements, as soon as the "Shooter Games" mode is active. Thus, all elements that are not connectable elements would become transparent for the eyes, and inactive for the mouse, to let the connectable objects do their job.

**Comment by ArgentArts** [comment]
Good point. Though, I still want a HIDE mode for other reasons, like hiding a roof so I can work on an interior better, etc. ;)

**Comment by UltraVox001** [comment]
Absoutely. It is as indispensable as the Wireframe mode.
